src/components/Table/DataTable/compenents/el-data-table/el-data-table.vue

```diff
@@ -865,6 +865,10 @@ export default {
     }
   },
   mounted() {
+    console.log('>>> Mounted: ', this.columns, this.columns.length)
+    setInterval(() => {
+      console.log('>>> Mounted: ', this.columns, this.columns.length)
+    }, 5000)
     if (this.saveQuery) {
       const query = queryUtil.get(location.href)
       if (query) {
```

Comment:
I would be happy to help with your code! Please post the relevant sections of your code and describe what you're trying to achieve so I can give more tailored feedback.